src/main/java/duke/taskmanagement/DukeException.java

```diff
@@ -0,0 +1,45 @@
+package duke.taskmanagement;
+
+
+public class DukeException {
+    protected String typeOfTask;
+
+    /**
+     * Create new duke exception to handle
```

Comment:
Perhaps the first sentence in the header comment starts with Creates more appropriate?